packages/manager/modules/storage/src/netapp/share/shares.routing.js

```diff
@@ -0,0 +1,12 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('netapp.share', {
+    url: '/share',
+    component: 'netappShare',
+    resolve: {
+      netappId: /* @ngInject */ ($transition$) =>
+        $transition$.params().netappId,
```

Comment:
If it is declared in the `netapp` state it will be inherited by all children state